types/node-telegram-bot-api/index.d.ts

```diff
@@ -155,6 +155,7 @@ declare namespace TelegramBot {
     interface SendMessageOptions extends SendBasicOptions {
         parse_mode?: ParseMode | undefined;
         disable_web_page_preview?: boolean | undefined;
+        entities?: MessageEntity[];
```

Comment:
```suggestion
        entities?: MessageEntity[] | undefined;
```

Note the other props here; this is for `exactOptionalPropertyTypes`.